bot_core.js

```diff
@@ -5,102 +5,208 @@ const bot = new Telegraf(token);
 const request = require(`request-promise`);
 const render = require(`./render-pool`);
 const atthemeEditorApi = require(`attheme-editor-api`);
-const { REGULAR_TEMPLATE, NEW_TEMPLATE } = require(`./preview-maker`);
-
+const {
+  MINIMALISTIC_TEMPLATE,
+  REGULAR_TEMPLATE,
+  NEW_TEMPLATE,
+} = require(`./preview-maker`);
 const RESEND_ON_ERRORS = [`RequestError`, `FetchError`];
 
 bot.context.downloadFile = async function (fileId) {
-    if (!fileId) {
-        fileId = this.message.document.file_id;
-    }
+  if (!fileId) {
+    fileId = this.callbackQuery.message.reply_to_message.document.file_id;
+  }
 
-    const file = await bot.telegram.getFile(fileId);
-    const fileContent = await request({
-        encoding: null,
-        uri: `http://api.telegram.org/file/bot${token}/${file.file_path}`,
-    });
+  const file = await bot.telegram.getFile(fileId);
+  const fileContent = await request({
+    encoding: null,
+    uri: `http://api.telegram.org/file/bot${token}/${file.file_path}`,
+  });
 
-    return fileContent;
+  return fileContent;
 };
 
 const handleStart = async (context) => {
-    const id = context.message.text.slice(`/start `.length).trim();
-
-    if (id.length === 0) {
-        await context.reply(`Send me an .attheme file to create its preview`);
-        return;
+  const id = context.message.text.slice(`/start `.length).trim();
+
+  if (id.length === 0) {
+    await context.reply(`Send me an .attheme file to create its preview`);
+    return;
+  }
+  const preview = await render({
+    theme,
+    name,
+    template: NEW_TEMPLATE,
+  });
+
+  const sendPreview = async () => {
+    try {
+      await context.replyWithPhoto(
+        { source: preview },
+        {
+          reply_to_message_id: context.message.message_id,
+          caption: `${name}\nCreated by @ThemePreviewBot`,
+        }
+      );
+    } catch (error) {
+      if (RESEND_ON_ERRORS.includes(error)) {
+        process.nextTick(sendPreview);
+      } else {
+        console.error(error);
+      }
     }
-    const { name, theme } = await atthemeEditorApi.downloadTheme(id);
-    const preview = await render({
+  };
+
+  sendPreview();
+};
+const choose = async (context) => {
+  const fileName = context.message.document.file_name;
+  if (fileName && fileName.endsWith(`.attheme`)) {
+    if (context.message.chat.type == "group") {
```

Comment:
```suggestion
    if (context.message.chat.type == `group`) {
```

also, this check ignores supergroups.